test/ecto/query/planner_test.exs

```diff
@@ -633,9 +634,18 @@ defmodule Ecto.Query.PlannerTest do
               [
                 {:inner, {"comments", Comment, 38_292_156, "world"}, true, ["join hint"]}
               ]},
-             {:from, {"posts", Post, 50_009_106, "hello"}, ["hint"]},
+             {:from, {"posts", Post, 132715331, "hello"}, ["hint"]},
              {:select, 1}
            ]
+
+   assert key == [:all,
+       {:lock, "foo"},
+       {:prefix, "foo"},
+       {:limit, {true, 1}},
+       {:where, [{:and, {:is_nil, [], [nil]}}, {:or, {:is_nil, [], [nil]}}]},
+       {:join, [{:inner, {"comments", Comment, 38292156, "world"}, true, ["join hint"]}]},
+       {:from, {"posts", Post, 132715331, "hello"}, ["hint"]},
+       {:select, 1}]
```

Comment:
```suggestion
```
it looks to me like this is an accidental duplicate but let me know if i'm mistaken